receiver/prometheusreceiver/internal/metricsbuilder.go

```diff
@@ -70,7 +74,6 @@ func (b *metricBuilder) AddDataPoint(ls labels.Labels, t int64, v float64) error
 		b.hasInternalMetric = true
 		lm := ls.Map()
 		delete(lm, model.MetricNameLabel)
-		b.logger.Infow("skip internal metric", "name", metricName, "ts", t, "value", v, "labels", lm)
```

Comment:
Why is this removed?

Reply:
It spams the logger every time a job is scraped - the internal metrics are expected and skipping them is the expected behavior so there's no need to log the event.